pipelines/build/prTester/README.md

```diff
@@ -1,123 +0,0 @@
-# PR Testers
```

Comment:
One last query, since we're keeping the linter and build jobs, can we keep the documentation in this repo? Could be copied and pasted into the root Readme?

Reply:
By keep documentation, I mean keep the linter and build parts but remove the other parts